torch/_inductor/ir.py

```diff
@@ -128,6 +127,18 @@ def reads_from_conv(buf, var_ranges):
     return False, None
 
 
+def convert_shape_to_symint(lst: List[Union[int, sympy.Expr]]) -> List[torch.SymInt]:
+    """
+    Takes a list of shapes from Inductor and converts them into symints (or just
+    ints if all shapes are static).
+    """
+    if all(isinstance(i, int) for i in lst):
+        return lst
+    if all(isinstance(i, sympy.Integer) for i in lst):
+        return [int(i) for i in lst]
```

Comment:
Hmm, I guess my fix for the type signature wasn't correct. Instead, I should have added int to the Union.